openfl/databases/tensor_db.py

```diff
@@ -11,6 +11,24 @@
 from openfl.utilities import TensorKey
 
 
+def weighted_average(tensors, weights):
```

Comment:
Could we use type hints to make it more clearer for user how to define such kind of functions?
```
def weighted_average(tensors: np.array, weights: np.array): -> np.array
```